### DISCUSSION TITLE
Tabellen Darstellung zerschossen

### DISCUSSION THREAD

**Discussion by bluebaer7** [OpenCCU/OpenCCU#1390] [🇩🇪 Fragen&Antworten] [answered]
Hallo, 

nutze Raspberrymatic auf einem Pi3b+ / CCU3  mit der Version 3.59.6.20210807 mit dem Twlight script V1.5 ([http://www.wikimatic.de/wiki/Datei:Twilight.tcl](url). 

Seit den letzten 2 Updates von Raspberrymatic ist die Tabellendarstellung im Firefox und Safari zerschossen. 

Bekannte Thema? 

LG Thomas 

![homematic](https://user-images.githubusercontent.com/44038795/130351267-f518f614-51fb-49c0-8051-abddb703b06e.PNG)

**Comment by jens-maus** [comment, marked as answer]
Bitte mal einen der aktuellen nightly snapshot builds probieren da es gut sein kann das diese bereits das Problem beheben weil an der Darstellung von html inhalten in systemvariablen verbesserungen umgesetzt wurden.

**Comment by bluebaer7** [comment]
Passt das Thema ist mit Version 3.59.6.20210911behoben. Danke.
